yjit/src/core.rs

```diff
@@ -1790,14 +1803,19 @@ fn gen_block_series_body(
     Some(first_block)
 }
 
+// Given an ISEQ pointer, convert PC to insn_idx
+fn iseq_pc_to_insn_idx(iseq: IseqPtr, pc: *mut VALUE) -> Option<u16> {
+    let pc_zero = unsafe { rb_iseq_pc_at_idx(iseq, 0) };
+    unsafe { pc.offset_from(pc_zero) }.try_into().ok()
+}
```

Comment:
Generally we put this kind of utility code into `yjit/src/cruby.rs`

Reply:
moved it in https://github.com/ruby/ruby/pull/7535/commits/e1b6553c67ce01ea29280db895d566ee9b93e4dd